src/ViewModels/CkanGroupViewModel.js

```diff
@@ -76,6 +77,14 @@ var CkanGroupViewModel = function(application) {
      */
     this.minimumMaxScaleDenominator = undefined;
 
+    /**
+     * Gets or sets any extra wms parameters that should be added to the wms query urls in this CKAN group.
+     * If this property is undefined or if {@link CkanGroupViewModel#filterByWmsGetCapabilities} then no parameters are added
```

Comment:
Ok, so if `filterByWmsGetCapabilities ` is true, we're going to do a `GetCapabilities` anyway, so we'll ignore `wmsParameters` and instead deduce the correct parameters based on what we see in `GetCapabilities`.  That's reasonable, but I don't see the code actually doing that.  Am I missing something?